librocksdb-sys/build.rs

```diff
@@ -2,9 +2,9 @@ extern crate cc;
 extern crate pkg_config;
 
 use pkg_config::probe_library;
-use std::process::Command;
-use std::fs::read_dir;
 use std::env::var;
+use std::fs::read_dir;
+use std::process::Command;
```

Comment:
How about using import groups? :upside_down_face:

Reply:
Maybe in another dedicated PR 😃 Or rustfmt will do this in the future for us. I really hope for it 😸